### PR TITLE
Initial commit for Composite module.

### PR DESCRIPTION
**Pull request by paperdigits**
docs for composite module

### CODE REVIEW

**Comment by alikstimix** [comment]
Couple of typos in first sentence: 
drom
ontop

And I think overlaid is preferable to overlayed

File: content/module-reference/processing-modules/composite.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
 ---
-title: composite (placeholder)
+title: composite
 id: composite
 weight: 10
 applicable-version: 4.8
 ---
 
+Drag and drop an image from the film strip onto the "drom image from filmstrip here" box to have the image overlayed ontop of the currently edited image. Then adjust the various placement and adjustment attributes of the image being overlayed.
```
Comment (elstoc) [inline content/module-reference/processing-modules/composite.md:8]: Could do with an initial brief description of the purpose of the module at the start without getting into implementation details. Perhaps something like "Overlay an already-processed image to create a composite of the two images." I'm not entirely satisfied with that sentence but it illustrates what I mean. Most modules have this initial description and it's useful as a quick reference.

Comment (paperdigits) [reply content/module-reference/processing-modules/composite.md]: Done.

**Comment by paperdigits** [comment]
@elstoc resolved your comments.

**Comment by elstoc** [comment]
One more typo but I'll resolve after merging

**Pull request merged**